homeWork2/getInfoPokemons.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+async function getInfoPokemons() {
+    const response = await fetch(`https://pokeapi.co/api/v2/pokemon`);
+    const objectResults = await response.json();
+    const arrayPokemonsUrl = objectResults.results.map(
+        (pokemon) => pokemon.url
+    );
+
+    const arrayPromises = arrayPokemonsUrl.map(async (url) => {
+        const response = await fetch(url);
+        const objectResults = await response.json();
+        return objectResults;
+    });
+
```

Comment:
1. you iterate twice. This stuff should be implemented at line 6
2. put async cb to `map`, `forEach`,` filter` is senseless. because promises are ignored inside of this methods
3. you can do the following: `const requests = data.map(dataItem => fetch().then(res) => res.json())`